integration_tests/tf_distribute_training_test.py

```diff
@@ -44,7 +44,10 @@ def test_model_fit():
     # Fit from numpy arrays:
     with strategy.scope():
         model.compile(
-            optimizer=optimizers.SGD(learning_rate=0.001, momentum=0.01),
+            # optimizer=optimizers.SGD(learning_rate=0.001, momentum=0.01),
```

Comment:
should we just delete this commented out line?

Reply:
Done. Thanks for catching this.